drivers/console/Kconfig

```diff
@@ -163,6 +163,34 @@ config RTT_TX_RETRY_IN_INTERRUPT
 
 endif
 
+config SEMIHOSTING_CONSOLE
+	bool
+	depends on ARM
+	prompt "Use semihosting console"
```

Comment:
Combine lines 167 and 169 into one: `bool "Use semihosting console"`. Do this for the other configs as well.